.gitmodules

```diff
@@ -1,9 +1,11 @@
 [submodule "spack"]
   path = spack
-  #url = https://github.com/spack/spack
-  #branch = develop
-  url = https://github.com/NOAA-EMC/spack
-  branch = jcsda_emc_spack_stack
+  ##url = https://github.com/spack/spack
+  ##branch = develop
+  #url = https://github.com/NOAA-EMC/spack
+  #branch = jcsda_emc_spack_stack
+  url = https://github.com/climbfuji/spack
```

Comment:
@AlexanderRichert-NOAA FYI this change is temporary so that the CI builds use the correct spack branch (a git submodule). Once the corresponding spack PR is merged, the author of the PRs (me in this case) will update the spack submodule pointer to point back to the head of noaa-emc/spack -> jcsda_emc_spack_stack and revert the change in `.gitmodules` before merging.